Generator/Package.swift

```diff
@@ -0,0 +1,28 @@
+// swift-tools-version: 5.9
```

Comment:
Since this is a new package, why not start with 6.1?

Reply:
Agreed, for the generator package I don't think we need to go below the current version.

Reply:
```suggestion
// swift-tools-version: 6.1
```

Reply:
Good catch, I've updated it to 6.1 here: https://github.com/swift-otel/swift-otel-semantic-conventions/pull/2/commits/bc1554177dab517615cea74c3b76c1b939e417fd